weather-module.js

```diff
@@ -0,0 +1,20 @@
+var http = require('http')
+module.exports = function(apiKey, city, callback){
+  city = encodeURIComponent(city)
+  http.get({
+      host: 'api.openweathermap.org',
+      path: `/data/2.5/forecast/?q=${city}&APPID=${apiKey}&units=imperial`
```

Comment:
Or maybe we show them `util.format` to give them more insight into a module they already know?